npm/qsharp/ux/circuit-vis/formatters/inputFormatter.ts

```diff
@@ -71,38 +71,15 @@ const formatInputs = (
 const _qubitInput = (y: number, subscript?: string): SVGElement => {
   const el: SVGElement = text("", leftPadding, y, 16);
 
-  // Create the main text node
-  const mainText = document.createElementNS(
-    "http://www.w3.org/2000/svg",
-    "tspan",
-  );
-  mainText.textContent = "|𝜓";
+  const subtext = subscript
+    ? `<tspan baseline-shift="sub" font-size="65%">${subscript}</tspan>`
+    : "";
 
-  // Create the subscript node if provided
-  if (subscript) {
-    const subscriptText = document.createElementNS(
-      "http://www.w3.org/2000/svg",
-      "tspan",
-    );
-    subscriptText.textContent = subscript;
-    subscriptText.setAttribute("baseline-shift", "sub");
-    subscriptText.setAttribute("font-size", "65%");
-    mainText.appendChild(subscriptText);
-  }
-
-  // Add the closing part of the text
-  const closingText = document.createElementNS(
-    "http://www.w3.org/2000/svg",
-    "tspan",
-  );
-  closingText.textContent = "⟩";
-
-  // Append all parts to the main SVG text element
-  el.appendChild(mainText);
-  el.appendChild(closingText);
+  el.innerHTML = `|<tspan class="qs-mathtext">ψ</tspan>${subtext}⟩</tspan>`;
```

Comment:
Could be fine, but this is another place to make sure we are using the specific characters we want for the bar and bracket.

Reply:
See from line 330 in util.ts. I checked all the files and made them consistent.

Reply:
We could use the mathChars table from utils.ts for the characters, including the psi.